### PR TITLE
Add Consumer Token Design: Type-I

### PR DESCRIPTION
**Pull request by taqtiqa-mark**
#### Project Abstract

To fund the preparation and submission to the SEC of a initial consumer token design (Type-I). Requesting a no-action letter permitting the offer and sale of such tokens/coins in a manner and under circumstances suitable for use in a permissioned blockchain, without registration under Section 5 of the Securities Act and Section 12(g) of the Exchange Act. The no-action letter request will be the first of three (possibly four) incremental submissions, culminating in a token design suited to operating a blockchain in a decentrailized/permissionless configuration. These later designs are the subject of subsequent grant funding rounds. The ultimate goal is to create a compliant token design that (**initially**) is licensed to and utilized by blockchain networks such as ~~Substrate.~~ Polkadot, Kusama and their network/ecosystem projects (hereafter ~~Sub-~~ Dotsama). In the intermediate state, operate under a model where the licensor (the grantee) seeks SEC approval for a token design, which is then available to be licensed to a licensee (e.g. Polkadot, Astar, etc.) for public issuance and sale. Offer a fixed and variable cost subscription model for the licensee. Subsequent token designs will attempt to: 1) obtain SEC approval that relaxes the licensing requirement, 2) make subscriptions optional (at the cost of sub-optimal efficiency/performance).

## Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [ ] **Level 2**:  Up to $30,000, 3 approvals
- [x] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [ ] Payment details have been provided (Polkadot AssetHub (DOT, USDC & USDT) address in the application and bank details via email, if applicable).
- [x] I understand that 30% of each milestone will be paid in vested DOT, to the Polkadot address listed in the application.
- [x] I am aware that, in order to receive a grant, I (and the entity I represent) have to successfully complete a KYC/KYB check.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by taqtiqa-mark** [comment]
This submission is sufficiently complete that it can be evaluated by the Web 3 Foundation council as to whether it is of interest to the W3F.  

If so I'm happy to consider any feedback and blockers etc. 

Also a note on how the quantum was arrived at (not sure if you want this in the application?)

Essentially I took the JAM prize as a guide. Since it seeks 3 implementations I divided the prize **roughly** by 3. Furthermore, while that is a prize for various implementations of existing tech, and not for anything novel, original, or value additive, I nonetheless decided against adding a novelty premium.

File: applications/consumer-token-type-i.md [hunk 1]
```diff
@@ -0,0 +1,235 @@
...
+Personal
+
+- <https://github.com/taqtiqa-mark>
+- <https://github.com/bbros-dev>
+- <https://github.com/taqtiqa>
+- <https://github.com/BegleyBrothers>
+
+### Team LinkedIn Profiles (if available)
+
+- <https://www.linkedin.com/in/tatiqa/>
```
Comment (semuelle) [inline applications/consumer-token-type-i.md:145]:
```suggestion
- <https://www.linkedin.com/in/taqtiqa/>
```

Comment (semuelle) [reply applications/consumer-token-type-i.md:145]: Hi @taqtiqa-mark. The URL provided does not exist. Can you fix the link, please?

**Review by takahser — CHANGES_REQUESTED** [review]
> Requesting a no-action letter permitting the offer and sale of such tokens/coins in a manner and under circumstances suitable for use in a permissioned blockchain

The W3F Grants program is focusing on open-source, rather than proprioritary  software. If you'd like to focus on token designs for permissioned blockchains I suggest you seek funding at the entity(-ies) that plan to use these permissoned blockchains instead.

 > Declining to award this grant project (rejecting the PR) will not prevent the Dotsama ecosystem members from licensing the initial and subsequent token designs. However, such licenses will only be available to the Dotsama ecosystem twenty (20) years after the date the first decentralized token design is licensed (timelines adopted from Polkadot patent). Currently, it is expected there will be three (3) incremental token designs that culminate in a token design suitable for use in a permissionless/decentralized network. Dotsama members can apply for licences before the twentieth year, provided they fully withdraw from and cease all activity in the Dotsama ecosystem, and do not participate in any non-compliant network/ecosystem, for the duration of their license.

This sounds like blackmail to me.

> 60,000 USD and 3,000,000 DOT

I don't understand why you'd think that this amount of funds is appropriate for this scope.

For these reasons, I don't think the proposal is suitable for our program and I've decided not to add my approval to it.

**Comment by taqtiqa-mark** [comment]
@takahser thank you for the prompt feedback. 

I write primarily to correct various falsehoods and misconceptions you have put forward:

> The W3F Grants program is focusing on open-source, rather than proprioritary software.

As a statement of fact this is patently false (pun intended): The W3F does exclusively focus on Polkadot, NPoS and DOT, ([30% of grants are forced to be vested as DOT](https://github.com/w3f/Grants-Program/blob/master/.github/pull_request_template.md?plain=1#L18)) and this software is patented to Parity, currently until 2038:

https://patents.google.com/patent/US20190058581A1/en

The no action letter will describe the token design, as will the application letter. I cannot make promises about, nor compel the SEC to release their letter under a specific license.  Once the design is public you can do what you want with it, including asking the SEC to approve your implementation of that design, or what ever variation of it you think is appropriate.

> This sounds like blackmail to me.

Understand, but seems likely due to a bad faith assumption, misunderstanding, misconception or some combination. The two problems being addressed are:

1. the free-rider problem
2. the incentive for incumbents to thwart innovation

***The token design is not patented, unlike the design of the W3F token, DOT,*** and so is able to be adopted/implemented by any chain (existing or new). I will be promoting one implementation, and that adoption incurs costs of making changes to their protocol, in the case of W3F this would be the Polkadot protocol. Again recall, unlike the W3F, this is not limited to people with informal or formal rights to use the Polkadot patent. 

This raises the free-rider problem: whereby a group (here the W3F/Dotsama) that does not contribute to the costs of the implementation can step in a take advantage of the setup costs bourne by the innovators/new entrants.  New entrants won't commit to bearing those costs if they know the incumbents (Dotsama) have a free option to step in and take all the benefits generated.

This free-rider and incumbency problem is particularly acute in this situation, for several reasons, but the most problematic (in my view) is:

-  The question of compliant token designs has been unnecessarily confused [due to the ***unsubstantiated claim*** of compliance by the W3F in asserting it has a token design that is compliant with set of consumer protections various courts have spelled out over nearly 80-years](https://polkadot.network/newsroom/press-releases/web3-foundation-announces-polkadot-blockchains-native-token-dot-has-morphed-and-is-software-not-a-security).  These consumer/investor protections evolved on a case by case basis in response to various scammers and hustlers and are not immaterial nor hypothetical

> If you'd like to focus on token designs for permissioned blockchains I suggest you seek funding at the entity(-ies) that plan to use these permissoned blockchains instead.

As I made very clear the permissioned design is an intremediate design, that will evolve to a permissionless/decentralized design. While the permissioned use case is valuable in it's own right, and is a phase a new chain could be expected to transition, it is not the end goal.

> I don't understand why you'd think that this amount of funds is appropriate for this scope.

I have already addressed this [here](https://github.com/w3f/Grants-Program/pull/2326#issuecomment-2152625196).  There are of course several ways one could arrive at a valuation, happy to hear your suggestions.

**Comment by taqtiqa-mark** [comment]
@semuelle, as you might appreciate there is currently a timetable that involves the SEC. 
Any information on the W3F time frames would be appreciated, e.g. when the relevant reviewers meet (monthly, weekly) etc.  

So far I have found the SEC staff to be accommodating, but I do not wish to exhaust that goodwill by rescheduling for no good reason. In particular @takahser's assumption of bad faith on my part (blackmail) has confirmed my initial skepticism that this proposal would be well received, or even received in good faith.

Appreciate any guidance you can give if you expect this process to extend beyond 30 June 2024.

**Comment by semuelle** [comment]
Hi @taqtiqa-mark. Thanks for the thorough responses. However, after careful consideration, we regret to inform you that we are unable to fund your project at this time. While we find your endeavour to be impressive, it does not closely align with the foundation's current strategic priorities and the level of support we typically provide to our grantees.

We still wish you best of luck and will follow any developments with interest.

**Comment by taqtiqa-mark** [comment]
Just recording here that the token design in question has reached a point where it is now clear that this grant, had it been made would have had to be unwound - courts and the SEC generally do not allow parties to retain any funds they received when they are aware the funds were obtained as part of an illegal scheme or arrangement. In fact there are instances where parties who were *unaware* of the nature of the scheme have had to return their funds.

**Pull request closed without merging**